djvm/src/main/kotlin/net/corda/djvm/rewiring/SandboxClassLoader.kt

```diff
@@ -63,8 +64,8 @@ class SandboxClassLoader private constructor(
      * Update the common bytecode cache with the classes we have generated.
      */
     override fun close() {
-        byteCodeCache.update(loadedByteCode)
         (parent as? SandboxClassLoader)?.close()
+        byteCodeCache.update(loadedByteCode)
```

Comment:
Was this change intentional? I can't see how it changes the logic.

Reply:
Flush the parent classloaders first, because the classes in this classloader may may depend on them. In practice I hope it _won't_ make any difference.